run_dir/design/deliveries.html

```diff
@@ -73,7 +73,7 @@ <h4>Project type
 	        {% end %}
         {% end %}
       </div>
-      <div class="form col-md-4 responsible-filters">
+      <div class="form col-md-3 responsible-filters">
         <h4>Bioinfo responsible
           <div class="btn-group">
             <a href="#" class="btn btn-sm btn-outline-dark all-responsibles">All</a>
```

Comment:
And some margin on line 94:

`<div class="delivery {{ ' '.join(project_status.get(project_id, [])) }} mt-5" id="bioinfo-delivery-project-{{ project_id }}">`